app/app.rb

```diff
@@ -0,0 +1,69 @@
+require 'sinatra/base'
+require "sinatra/json"
+require './lib/menu'
+require './lib/order_list'
+require './lib/order'
+require './lib/tax'
+require './lib/discount'
+require './lib/receipt'
+require './lib/payment'
+require './app/modules/my_helpers'
+
+
+class TillTechTest < Sinatra::Base
+  set :root, File.dirname(__FILE__)
+  helpers Sinatra::JSON
+  include MyHelpers
+  set :static, true
+  
+  set :order_list, OrderList.new
+  set :menu      , Menu.new([{name: 'Cafe Latte', price: 2.5 },
```

Comment:
Unnecessary spacing detected.<br>Space found before comma.<br>Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.